drivers/led/aw20216s.h

```diff
@@ -54,6 +54,10 @@
 #define g_aw_leds g_aw20216s_leds
 // ========
 
+#if defined(RGB_MATRIX_AW20216S)
+#    define AW20216S_LED_COUNT RGB_MATRIX_LED_COUNT
+#endif
```

Comment:
Should we establish a pattern like this?
```suggestion
#if defined(RGB_MATRIX_AW20216S) && !defined(AW20216S_LED_COUNT)
#    define AW20216S_LED_COUNT RGB_MATRIX_LED_COUNT
#endif
```

Reply:
Shouldn't be any need to. For `RGB_MATRIX_DRIVER=aw20216s` the LED count will be inferred with this code. For `RGB_MATRIX_DRIVER=custom` that uses the AW20216S driver, or just using it manually, you would need to define the LED count.